tests/jsunit/event_test.js

```diff
@@ -238,7 +238,7 @@ function test_move_constructorCoordinate() {
   setUpMockMethod(mockControl_, Blockly.utils, 'genUid', null, ['1', '2']);
   try {
     var block1 = createSimpleTestBlock(workspace);
-    var coordinate = new goog.math.Coordinate(3,4);
+    var coordinate = new Blockly.utils.Coordinate(3,4);
```

Comment:
Not the fault of this PR, but can we get some spaces between arguments here and below?